.github/workflows/extract-translation-source-files.yml

```diff
@@ -186,10 +187,12 @@ jobs:
           cd translations/${{ matrix.repo }}
           # remove translations/${{ matrix.repo }}/.git so we don't commit a submodule
           rm -rf .git
+          # find transifex_input.json
+          TRANSIFEX_JSON_PATH=$(find . -name 'transifex_input.json')
```

Comment:
Is it worth doing this in regex, like we did for the python-based repos:
```
EN_DIR=$(find . -type d -regex ".+conf\/locale\/en$")
```

Reply:
i think keeping it simple and just using the name makes sense here. to the best of my knowledge, every js based repo has the name `transifex_input.json` in common.

if we were looking for more than just the one file a regex would make a lot of sense